### ISSUE TITLE
Feature request: add properties and onChange and onSave methods to report list of chips

### ISSUE THREAD

**Issue by davoutuk** [BruckCode/chips_input_autocomplete#3]
Hello,

I'd like to request the following feature improvement.  

**Business driver**
Having a widget that allows the data entry of multiple chips but where the widget takes an input/output of a single String value seems at odds with the most expected use case.

As the widget is being used to capture the input of multiple chips,  the expectation would be that the 'initialValue' and 'onSaved' properties would be anchored around a "List<String>" rather than a String. 

**Suggested improvement:**
Add new properties to support the widget as a collector for a list of chip strings.

New property:
`List<String> initialChips;`

New event handlers:
```
onValidateChips: Function(List<String>)
onChangedChips : Function(List<String>?>
onSavedChips: Function(List<String>?)

```
Widget parameter validation:
Throw an exception if both 'initialValue' and 'initialChips' are both set
Throw an exception if both 'onSaved' and 'onSavedChips' are both set
Throw an exception if both the 'onChanged' and 'onChangedChips' are both set.

**Comment by davoutuk** [comment]
I see that you have made the request feature change.  When is this going live on pub.dev?

**Comment by EmilyMoonstone** [comment]
I just published version 1.2.1. Thank you for your feature request.
